### PR TITLE
Improve event builder names

### PR DESCRIPTION
**Pull request by AlanGriffiths**
We already broke ABI, so now is a good time

### CODE REVIEW

File: include/client/mir/events/event_builders.h [hunk 1]
```diff
@@ -41,20 +41,20 @@ namespace mir
...
-EventUPtr make_event(frontend::SurfaceId const& surface_id, MirWindowAttrib attribute, int value);
+EventUPtr make_window_configure_event(frontend::SurfaceId const& surface_id, MirWindowAttrib attribute, int value);
 // Close surface event
-EventUPtr make_event(frontend::SurfaceId const& surface_id);
+EventUPtr make_window_close_event(frontend::SurfaceId const& surface_id);
 // Keymap event
-EventUPtr make_event(frontend::SurfaceId const& surface_id, MirInputDeviceId id, std::string const& model,
-                     std::string const& layout, std::string const& variant, std::string const& options);
+EventUPtr make_keymap_event(frontend::SurfaceId const& surface_id, MirInputDeviceId id, std::string const& model,
+                            std::string const& layout, std::string const& variant, std::string const& options);
```
Comment (wmww) [inline include/client/mir/events/event_builders.h:55]: Good time to also use current formatting?

File: src/server/input/surface_input_dispatcher.cpp [hunk 2]
```diff
@@ -269,7 +270,7 @@ bool dispatch_scene_change_enter_exit_events(
                 mir_pointer_event_axis_value(ctx.pev, mir_pointer_axis_y),
                 0, 0, // No scrolling
                 0, 0  // No relative motion
-            );
+                                                      );
```
Comment (wmww) [inline src/server/input/surface_input_dispatcher.cpp:273]: lol

**Review by wmww — CHANGES_REQUESTED** [review]
Just formatting

File: src/client/events/event_builders.cpp [hunk 3]
```diff
@@ -153,9 +153,9 @@ mir::EventUPtr mev::make_event(frontend::SurfaceId const& surface_id, geometry::
     return make_uptr_event(e);
 }
 
-mir::EventUPtr mev::make_event(MirInputDeviceId device_id, std::chrono::nanoseconds timestamp,
-    std::vector<uint8_t> const& cookie, MirKeyboardAction action, xkb_keysym_t keysym,
-    int scan_code, MirInputEventModifiers modifiers)
+mir::EventUPtr mev::make_key_event_event(MirInputDeviceId device_id, std::chrono::nanoseconds timestamp,
+                                         std::vector<uint8_t> const& cookie, MirKeyboardAction action, xkb_keysym_t keysym,
+                                         int scan_code, MirInputEventModifiers modifiers)
```
Comment (wmww) [inline src/client/events/event_builders.cpp:158]: More formatting needed in this file

File: src/server/input/validator.cpp [hunk 4]
```diff
@@ -160,13 +160,13 @@ void mi::Validator::handle_touch_event(MirEvent const& event)
...
-                    mir_touch_event_modifiers(touch_event),
-                    contacts
-                    ));
+            dispatch_valid_event(*mev::make_touch_event(
+                id,
+                std::chrono::nanoseconds{mir_touch_event_modifiers(touch_event)},
+                std::vector<uint8_t>{},
+                mir_touch_event_modifiers(touch_event),
+                contacts
+                                                       ));
```
Comment (wmww) [inline src/server/input/validator.cpp:169]: And here

File: src/server/scene/surface_event_source.cpp [hunk 5]
```diff
@@ -43,39 +43,40 @@ ms::SurfaceEventSource::SurfaceEventSource(
...
-        ));
+        event_sink->handle_event(
+            mev::make_window_output_event(
+                id,
+                new_output_properties->dpi,
+                new_output_properties->scale,
+                new_output_properties->refresh_rate,
+                new_output_properties->form_factor,
+                static_cast<uint32_t>(new_output_properties->id.as_value())
+                                         ));
```
Comment (wmww) [inline src/server/scene/surface_event_source.cpp:62]: And here

**Review by wmww — CHANGES_REQUESTED** [review]
More formatting

**Review by wmww — APPROVED** [review]
bors delegate+

**Comment by AlanGriffiths** [comment]
bors r=wmww

**Pull request merged**